transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -71,6 +93,105 @@ protected AbstractEpollUnsafe newUnsafe() {
     }
 
     /**
+     * @see {@link #spliceTo(AbstractEpollStreamChannel, int, ChannelPromise)}
+     */
+    public final ChannelFuture spliceTo(final AbstractEpollStreamChannel ch, final int len) {
+        return spliceTo(ch, len, newPromise());
+    }
+
+    /**
+     * Splice from this {@link AbstractEpollStreamChannel} to another {@link AbstractEpollStreamChannel}.
+     *
+     * Please note:
+     * <ul>
+     *   <li>both channels need to be registered to the same {@link EventLoop}, otherwise an
+     *   {@link IllegalArgumentException} is thrown. </li>
+     *   <li>{@link EpollChannelConfig#getEpollMode()} must be {@link EpollMode#LEVEL_TRIGGERED} for this and the
+     *   target {@link AbstractEpollStreamChannel}</li>
+     * </ul>
+     *
+     */
+    public final ChannelFuture spliceTo(final AbstractEpollStreamChannel ch, final int len,
+                                        final ChannelPromise promise) {
+        if (ch.eventLoop() != eventLoop()) {
+            throw new IllegalArgumentException("EventLoops are not the same");
+        }
+        if (len < 0) {
+            throw new IllegalArgumentException("len must be >= 0 but was " + len);
+        }
+        if (ch.config().getEpollMode() != EpollMode.LEVEL_TRIGGERED
+                || config().getEpollMode() != EpollMode.LEVEL_TRIGGERED) {
+            throw new IllegalStateException("spliceTo(...) only supported when using "
+                    + EpollMode.LEVEL_TRIGGERED.name());
+        }
+        checkNotNull(promise, "promise");
+        if (!isOpen()) {
+            promise.tryFailure(CLOSED_CHANNEL_EXCEPTION);
+        } else {
+            SpliceInTask task = new SpliceInChannelTask(ch, len, checkNotNull(promise, "promise"));
+            spliceQueue.add(task);
+            failSpliceIfClosed(promise);
+        }
+        return promise;
+    }
+
+    /**
+     * @see {@link #spliceTo(FileDescriptor, int, int, ChannelPromise)}
```

Comment:
Please fill the API documentation properly. `@see` doesn't help a lot when a user wants to quick-browse the documentation.